setup.py

```diff
@@ -40,6 +40,7 @@
     packages=['databricks', 'databricks.koalas', 'databricks.koalas.missing'],
     extras_require={
         'spark': ['pyspark>=2.4.0'],
+        'mlflow': ['mlflow>=0.2.0'],
```

Comment:
@thunterdb, can we increase the minimal version to almost the latest ones? I was thinking people will mostly use the latest version of mlflow